config/projects.yml

```diff
@@ -264,6 +264,7 @@ testing:
       docs_path: 'components/testing'
       deprecation_warnings: "/.*each.*/"
       php: ['7.1', '7.2']
+      phpunit_version: 6.5
```

Comment:
do we have something, what renders that information to the right places? You should edit the templates too then.

Reply:
i expected this to be translated to https://github.com/symfony-cmf/dev-kit/blob/f114ea72544b31d49115a0aae1f686712834520d/project/.travis.yml.twig#L41 - am i wrong with that assumption?

Reply:
Could be correct, can your run it for testing lokal without pushing it?

Reply:
unfortunately not: `./dev-kit dispatch --with-files testing`

```
Files for master
----------------

diff --git a/composer.json b/composer.json
index de4cb3b..085c9ef 100644
--- a/composer.json
+++ b/composer.json
@@ -18,22 +18,22 @@
         "friendsofsymfony/jsrouting-bundle": "^1.1 || ^2.1",
         "jackalope/jackalope": "^1.3",
         "jackalope/jackalope-doctrine-dbal": "^1.3",
-        "symfony/browser-kit": "^2.8 || ^3.0 || ^4.0",
-        "symfony/console": "^2.8 || ^3.0 || ^4.0",
-        "symfony/css-selector": "^2.8 || ^3.0 || ^4.0",
-        "symfony/debug": "^2.8 || ^3.0 || ^4.0",
-        "symfony/dependency-injection": "^2.8 || ^3.0 || ^4.0",
-        "symfony/doctrine-bridge": "^2.8 || ^3.0 || ^4.0",
-        "symfony/framework-bundle": "^2.8.1 || ^3.0 || ^4.0",
-        "symfony/http-foundation": "^2.8 || ^3.0 || ^4.0",
-        "symfony/http-kernel": "^2.8 || ^3.0 || ^4.0",
-        "symfony/monolog-bundle": "^3.1",
-        "symfony/process": "^2.8 || ^3.0 || ^4.0",
-        "symfony/security-bundle": "^2.8 || ^3.0 || ^4.0",
-        "symfony/twig-bundle": "^2.8 || ^3.0 || ^4.0"
+        "symfony/browser-kit": "^2.8 || ^3.3 || ^4.0",
+        "symfony/console": "^2.8 || ^3.3 || ^4.0",
+        "symfony/css-selector": "^2.8 || ^3.3 || ^4.0",
+        "symfony/debug": "^2.8 || ^3.3 || ^4.0",
+        "symfony/dependency-injection": "^2.8 || ^3.3 || ^4.0",
+        "symfony/doctrine-bridge": "^2.8 || ^3.3 || ^4.0",
+        "symfony/framework-bundle": "^2.8 || ^3.3 || ^4.0",
+        "symfony/http-foundation": "^2.8 || ^3.3 || ^4.0",
+        "symfony/http-kernel": "^2.8 || ^3.3 || ^4.0",
+        "symfony/monolog-bundle": "~3.1",
+        "symfony/process": "^2.8 || ^3.3 || ^4.0",
+        "symfony/security-bundle": "^2.8 || ^3.3 || ^4.0",
+        "symfony/twig-bundle": "^2.8 || ^3.3 || ^4.0"
     },
     "require-dev": {
-        "symfony/phpunit-bridge": "^4.0"
+        "symfony/phpunit-bridge": "^3.3 || ^4.0"
     },
     "autoload": {
         "psr-4": {
```

Reply:
Shall i do that for you on our "master"?

Reply:
if you have the time to do so, yes please